tools/asciitable/main.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type ParsedResult struct {
+	Conclusion string `json:"conclusion"`
+	Cap        string `json:"cap"`
+	URL        string `json:"html_url"`
+}
+
+type ResultsMap map[string][]ParsedResult
+
+func readJSONFile(filePath string) ([]byte, error) {
+	return os.ReadFile(filePath)
+}
+
+func parseJSON(jsonData []byte, namedKey string) ([]ParsedResult, error) {
+	var jobs []ParsedResult
+	if namedKey == "" {
+		err := json.Unmarshal(jsonData, &jobs)
+		return jobs, err
+	}
+
+	var results ResultsMap
+	err := json.Unmarshal(jsonData, &results)
+	if err != nil {
+		return nil, err
+	}
+
+	if val, ok := results[namedKey]; ok {
+		return val, nil
+	}
+
+	return jobs, nil
+}
+
+func calculateColumnWidths(firstColumnHeader, secondColumnHeader string, sections map[string][]ParsedResult) (int, int) {
+	maxFirstColumnLen := len(firstColumnHeader)
+	maxSecondColumnLen := len(secondColumnHeader)
+
+	for section, jobs := range sections {
+		if len(section) > maxFirstColumnLen {
+			maxFirstColumnLen = len(section)
+		}
+		for _, job := range jobs {
+			if len(job.Cap) > maxFirstColumnLen {
+				maxFirstColumnLen = len(job.Cap)
+			}
+		}
+	}
+
+	return maxFirstColumnLen, maxSecondColumnLen
+}
+
+func writeResultsToFile(fileName string, firstColumnHeader, secondColumnHeader, currentSection string, jobs []ParsedResult) error {
```

Comment:
Can we use https://github.com/olekukonko/tablewriter instead? Okay if we need it urgently but seems like we can replace this code with a more solid library

Reply:
will create a low prio ticket, wdyt?